packages/mobile/components/ArticleContent.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from "react";
+import { useState, useEffect } from "react";
+
+import styled from "styled-components";
+import { View, Text, SafeAreaView, Button, TextInput } from "react-native";
+import { ActivityIndicator } from "react-native-paper";
+import moment from "moment";
+import Breadcrumbs from "./Breadcrumbs";
+import ArticleEditor from "./ArticleEditor";
+
+import { useArticleQuery } from "../local_core/generated/graphql";
+
+const StyledSafeAreaView = styled(SafeAreaView)`
+  flex: 1;
+  background: #fff;
+`;
+
+const StyledText = styled(Text)`
+  width: 335px;
+  height: 42px;
+  left: 20px;
+
+  font-size: 12px;
+  line-height: 20px;
+
+  display: flex;
+  align-items: center;
+  text-align: center;
+
+  color: #bdbdbd;
+`;
+
+const TitleEditText = styled(TextInput)`
+  width: 100%;
+  padding-top: 36px;
+  left: 19px;
+  font-weight: bold;
+  font-size: 48px;
+  letter-spacing: -1.5px;
+  color: rgba(0, 0, 0, 0.87);
+`;
+
+const StyledLoadingView = styled(View)`
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+interface Props {
+  route: any;
+  navigation: any;
+}
+
+const ArticleContent = ({ route, navigation }: Props) => {
+  const { data, loading, error } = useArticleQuery({
+    variables: { id: route.params.articleId },
+  });
+
+  // const [updateArticle] = useMutation(UPDATE_ARTICLE_MUTATION);
```

Comment:
Please try to avoid commiting comments, I won't comment all of them but there are plenty. I understand this is because you're trying to be more organized with the commits and removed on the fly some functionalities and it's easier to leave them commented, just take it as a note for the future